consensus/bundle_validator/tests/bundle_validator.c

```diff
@@ -34,6 +34,12 @@ void tearDown() {
   TEST_ASSERT(iota_consensus_bundle_validator_destroy() == RC_OK);
 }
 
+void test_prepare_4txs(iota_transaction_t *txs) {
```

Comment:
Can you make it static ?

Reply:
Yes, thank you for pointing this out. test_prepare_4txs() may be a common name.
#466c03e